### ISSUE TITLE
Python 3 by default?

### ISSUE THREAD

**Issue by kaedroho** [torchbox/wagtail-template#22]
Wagtail and its dependencies all have good Python 3 support and the Python 3 world is now in quite good shape too. Python 3 gives a number of minor syntax improvements and some useful new features as well.

So I think it's worth asking. Should we switch the template to Python 3 by default?

**Comment by gasman** [comment]
Sounds good to me, provided it doesn't cause any complications for live server deployments (and I don't see why it would, since it's just a package install away on any sane Linux distro). @ftarnell?

**Comment by ftarnell** [comment]
This is fine for Tbx because we have a locally-compiled Python 3.4 now, but Debian 7 (stable) doesn't include 3.4, only 3.2, and it doesn't seem to be in backports either, so this might be a problem for some third-party users.

**Comment by zerolab** [comment]
FWIW, wagtail-template is our own, opinionated starter. So there should not be any issues judging from the above.

**Comment by gasman** [comment]
Yep, that sounds fine, cheers. Unlike [wagtail's built-in project template](https://github.com/torchbox/wagtail/tree/master/wagtail/project_template), wagtail-template isn't trying to be all things to all people, so it's appropriate enough for us to specify Python 3, and let people fork if they don't like it :-)
